### PR TITLE
docs: add config merge to faq

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: docs/source/zh-cn/faq.md [hunk 1]
```diff
@@ -3,6 +3,13 @@ title: 常见问题
 
 如果下面的内容无法解决你的问题，请查看 [egg issues](https://github.com/eggjs/egg/issues)。
 
+## 配置合并排查
```
Comment (popomore) [inline docs/source/zh-cn/faq.md]: 这里写问句？

Comment (atian25) [inline docs/source/zh-cn/faq.md]: 这句话没有疑问语气吧?  除非是 `如何知道最终的配置结果?`

Comment (popomore) [inline docs/source/zh-cn/faq.md]: FAQ 最好都用问句吧

Comment (atian25) [inline docs/source/zh-cn/faq.md]: 2 个 title 都改了, 再看看

**Pull request merged**